nxdrive/direct_edit.py

```diff
@@ -652,9 +652,11 @@ def _unlock(self, remote: Remote, uid: str, /) -> bool:
         except NotFound:
             return True
         except HTTPError as exc:
-            if exc.status is codes.INTERNAL_SERVER_ERROR:
+            if exc.status == codes.INTERNAL_SERVER_ERROR:
                 # INTERNAL_SERVER_ERROR is raised on double lock.
-                return True
+                locked = re.findall(r"Document already locked by", exc.message)
```

Comment:
No need for a regexp:
```suggestion
                if "Document already locked by" in exc.message:
                    return True
```